openfl/interface/workspace.py

```diff
@@ -232,6 +234,11 @@ def certify():
     echo('1.  Create Root CA')
     echo('1.1 Create Directories')
 
+    if cert_path:
+        CERT_PATH = Path(cert_path).absolute()
+        (CERT_PATH / 'cert').mkdir(parents=True, exist_ok=True)
+        CERT_DIR = CERT_PATH/ 'cert' # NOQA
```

Comment:
Could we avoid reassigning the constant value imported from another module?

Reply:
`CERT_DIR` path is either fixed (`Path('cert')`) or provided by the user at runtime now. Avoiding reassigning this variable might be difficult.

Reply:
Try introducing a local variable